packages/js/email-editor/package.json

```diff
@@ -46,6 +46,7 @@
 		"lint:fix:lang:css": "pnpm lint:css-fix",
 		"lint:lang:js": "pnpm lint:js",
 		"lint:lang:css": "pnpm lint:css",
+		"lint:lang:types": "pnpm check-types",
```

Comment:
I think we can just drop `check-types` and declare `"lint:lang:types": "NODE_OPTIONS=--max_old_space_size=2048 tsc --noEmit",` for simplicity.

Reply:
Thanks for the suggestion, updated the PR.